empiric-package/empiric/core/mixins/oracle.py

```diff
@@ -48,18 +48,22 @@ async def publish_spot_entry(
         return invocation
 
     async def publish_many(
-        self, entries: List[SpotEntry], pagination=80, max_fee=int(1e18)
+        self,
+        entries: List[SpotEntry],
+        pagination: Optional[int] = 80,
+        max_fee=int(1e18),
     ) -> List[InvokeResult]:
         if len(entries) == 0:
             logger.warning("Skipping publishing as entries array is empty")
             return
 
         invocations = []
+        serialized_entries = SpotEntry.serialize_entries(entries)
```

Comment:
feels surprising that `serialize_entries` would filter as well, maybe we should make that a separate step? Also we will need to add logic to publish `FutureEntry`s which we currently use for the yield curve

Reply:
publishing FutureEntry will be a refactor.  serialize_entries filters out invalid entries, since a large number of them fail.  We could probably filter before we serialize but this ended up being convenient since a lot of the requests from publishers come back invalid.

Reply:
Good to filter them, and ok to filter when serializing. So the plan is to do a separate PR to publish the Future entries?

Reply:
yeah this is just a small fix to update the pagination.  we can handle that once we support publishing FuturEntries